bin/brew

```diff
@@ -161,7 +168,7 @@ then
   # Hide shellcheck complaint:
   # shellcheck source=/dev/null
   source "$HOMEBREW_BASH_COMMAND"
-  { "$HOMEBREW_COMMAND" "$@"; exit $?; }
+  { "brew-$HOMEBREW_COMMAND" "$@"; exit $?; }
```

Comment:
Thanks. It shouldn't feel too much off, given our external commands also have to be named `brew-<command>` and our internal Ruby commands are `Homebrew.<command>` (thus also have a prefix). This definitely has the benefit that we won't run into any collisions with shell keywords or shell built-ins, though I hope/assume we don't plan to reimplement major parts of Homebrew in shell. :grinning: